modules/objdetect/include/opencv2/objdetect/face.hpp

```diff
@@ -90,6 +90,22 @@ class CV_EXPORTS_W FaceDetectorYN
                                               int top_k = 5000,
                                               int backend_id = 0,
                                               int target_id = 0);
+
+    /** @overload
+     * @param framework Name of origin framework.
+     * @param bufferModel A buffer with a content of binary file with weights.
+     * @param bufferConfig A buffer with a content of text file contains network configuration.
+     */
+    CV_WRAP static Ptr<FaceDetectorYN> create(const String& framework,
```

Comment:
Do you export the face detection onnx model to some other framework formats that have seperated weight and net arch?

Reply:
No, I just follow `readNet` api for consistency.

Reply:
Ok, thanks.